test/quantization/test_bias_correction.py

```diff
@@ -0,0 +1,98 @@
+import torch
+import torch.nn as nn
+from torch.testing._internal.common_quantization import QuantizationTestCase
+from torch.quantization import QuantStub, DeQuantStub
+import torch.quantization._correct_bias as _correct_bias
```

Comment:
nit: I think you can just have `_` for the filename and remove `_` for the function name so that we don't need to change the callsite when we make this public